src/vs/workbench/contrib/webview/browser/baseWebviewElement.ts

```diff
@@ -312,7 +312,11 @@ export abstract class BaseWebview<T extends HTMLElement> extends Disposable {
 	}
 
 	protected style(): void {
-		const { styles, activeTheme, themeLabel } = this.webviewThemeDataProvider.getWebviewThemeData();
```

Comment:
Instead of adding a new transform function on the `option`, could you pass in a custom `webviewThemeDataProvider` that does this transform?